src/AnalysisExporter/TemplateInstantiations/TemplateInstantiationsExporter.cpp

```diff
@@ -39,15 +39,25 @@ bool TemplateInstantiationsExporter::ExportTo(const std::string& path) const
     // aggregate data
     typedef std::unordered_map<std::string, DataPerTemplate> TAggregatedData;
     TAggregatedData aggregatedData;
+    std::string templateName = "Unknown";
+
     for (auto&& pair : m_templateInstantiations)
     {
         const std::chrono::nanoseconds& timeElapsed = pair.second.Duration;
 
         auto itSpecializationTemplateName = m_symbolNames.find(pair.second.Specialization);
-        assert(itSpecializationTemplateName != m_symbolNames.end());
+        
+        if (itSpecializationTemplateName != m_symbolNames.end())
+        {
+            templateName = itSpecializationTemplateName->second;
+        }
+        else
+        {
+            templateName = "Unknown";
+        }
```

Comment:
How about not registering the `Unknown` template name at all and switching to this?

```
auto itSpecializationTemplateName = m_symbolNames.find(pair.second.Specialization);
if (itSpecializationTemplateName == m_symbolNames.end())
{
    // a TemplateInstantiation activity has been recorded but its related SymbolName hasn't been emitted
    // we can't know which template it refers to, so we can only skip it
    continue;
}
```

After all, it won't be correctly registered in the `BuildTimeline.json` report either.
And this way users won't have an entry listed `Unknown` with the accumulated time of (probably) unrelated templates.